decidim-results/app/views/decidim/results/results/show.html.erb

```diff
@@ -34,6 +37,7 @@
         </div>
       </div>
     </div>
+    <%= render partial: "decidim/shared/share_modal" %>
```

Comment:
I think we should refactor the share code into a helper or something where you can include the:

- `content_for` things
- The partial rendering
- The javascript and stylesheet links

Maybe something like `social_share_for(model)`. With that we can reduce the amount of code.